.env

```diff
@@ -76,6 +76,7 @@ ARROW_R_DEV=TRUE
 R_IMAGE=ubuntu-gcc-release
 R_ORG=rhub
 R_PRUNE_DEPS=FALSE
+R_DUCKDB_DEV=FALSE
```

Comment:
can we do this without adding to .env? Like, invoke docker (compose) with `-e R_DUCKDB_DEV=TRUE` in the job you care about? we don't need this as a parameter to docker-compose, just as an env var in the job.